### PR TITLE
Tjstogoski contrib

### PR DESCRIPTION
**Pull request by tjstogoski**
Uploaded a portfolio post about Banshee Reeks Nature Preserve

Adresses Issue #175

### CODE REVIEW

File: posts/01-first-map/tjstogoski/brnp-map.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Banshee Reeks Nature Preserve, the hidden treasure of Northern Virginia
+
+Banshee Reeks Nature Preserve (BRNP) protects 695 acres of the Northern Virginia Piedmont through a conservation easement held by the Virginia Outdoors Foundation. It is managed by the Department of Parks, Recreation & Community Services of Loudoun County, Virginia, USA. The map shows the forests, fields, wetlands, and waters that make up the preserve and serve to teach us of the natural and cultural resources of the region.
```
Comment (VictoriAragon) [inline posts/01-first-map/tjstogoski/brnp-map.md:3]: This is a great description of the area. This works for the assignment @nquarder

**Review by VictoriAragon — COMMENTED** [review]
This is a great start and provides what is needed. Maybe a reference needed? @nquarder

**Review by ZimmZone — DISMISSED** [review]
Looks good, I like the map and subject. I echo the suggestion to add a reference for the text if you have it.

File: posts/01-first-map/tjstogoski/brnp-map.md (hunk 1 above)
Comment (nquarder) [reply posts/01-first-map/tjstogoski/brnp-map.md:3]: Yes agreed, @VictoriAragon - @tjstogoski when you get a second try adding a hyperlink to BRNP or another citation of some kind.

**Comment by tjstogoski** [comment]
@VictoriAragon @ZimmZone Thank you for the feedback. I added a hyperlink to the location website.

**Comment by eculler** [comment]
Thanks @tjstogoski , nice work!

**Pull request merged**